pkg/elemental/models/elemental.cattle.io.machineregistration.js

```diff
@@ -15,8 +15,12 @@ const DEFAULT_CREATION_YAML = `config:
       passwd: root
   elemental:
     install:
-      poweroff: true
-      device: /dev/nvme0n1`;
+      reboot: true
+      device: /dev/nvme0n1
```

Comment:
should we move to `device-selector` here to select the "first" disk ? 🤔

Reply:
I was also wondering about it and I do not have clear preference. I was wondering about it, but which selector criteria should we use? A list of common device names? First disk with at least certain size (the minimum according to our default partition sizes)?

Reply:
done